tests/SourceTest.php

```diff
@@ -2,211 +2,98 @@
 
 namespace Stripe;
 
-class SourceTest extends TestCase
+define('TEST_RESOURCE_ID', 'src_123');
+
+class SourceTest extends StripeMockTestCase
```

Comment:
@ob-stripe Flagging that `Source.php` has the `all` method even though the API never supported it. Did not add a test since this should not exist in the first place I think

Reply:
👍  Let's fix this in a separate PR.